extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -965,7 +965,7 @@ private void fetchAndroidMobileJsonPlayer(@Nonnull final ContentCountry contentC
                         .value(RACY_CHECK_OK, true)
                         // Workaround getting streaming URLs which return 403 HTTP response code by
                         // using some parameters for Android client requests
-                        .value("params", "CgIQBg")
+                        .value("params", "CgIIAQ%3D%3D")
```

Comment:
Can you explain this change?

Reply:
See https://github.com/TeamNewPipe/NewPipeExtractor/pull/1168#issuecomment-2059827022.